static/index.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Themeroller</title>
```

Comment:
Umm @arthurvr , me and srisk had a conversation about that on IRC, I needed this environment PR for splitting things into two PRs UI and API and laying a base, that page will soon get replaced by UI page in next PR